analyze.sh

```diff
@@ -132,6 +132,9 @@ fi
 
 echo "resultsTable: $resultsTable"
 
+# delete reuslts.md
+rm -f results.md 
```

Comment:
Why are we deleting this file?

Reply:
because when we use matrix to run the script individually i noticed it doesn't actually delete the "readme.md"... 
**it does deleted it buh it only base on the service instant that's current running...** 

the only way to delete it is through the **analyse.sh**